taxi/test/test_forms.py

```diff
@@ -0,0 +1,92 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.test import TestCase
```

Comment:
Add one blank line between different import groups in all places in the project to follow PEP-8